.github/workflows/CI.yml

```diff
@@ -110,6 +122,25 @@ jobs:
             --tlapm_lib_path deps/tlapm/library                     \
             --community_modules_jar_path deps/community/modules.jar \
             --manifest_path manifest.json
+      - name: Get proof fingerprint cache
+        if: matrix.os != 'windows-latest'
+        id: cache-proofs-restore
+        uses: actions/cache/restore@v3
+        with:
+          path: .tlacache
+          key: ${{ runner.os }}-proof-fingerprints
+      - name: Check proofs
+        if: matrix.os != 'windows-latest'
+        run: |
```

Comment:
Does the CI time out without old fingerprints? If not, the extra complexity might not be warranted.

Reply:
Not currently, but once some of the currently-commented-out proofs are included the CI time will increase by quite a bit (probably 5-10 minutes).

Reply:
IMO it is better to catch regression at the expense of increased CI times, rather than to distributed broken examples.